### DISCUSSION TITLE
GitHub Achievement Discussions

### DISCUSSION THREAD

**Discussion by PurpleMonkey729** [PurpleMonkey729/GitHub-Profile-Achievement-Badge#1] [Announcements]
Welcome to GitHub Achievement Discussion.
Here, we can discuss how to get GitHub Achievements.
One importance is that you can get all GitHub achievements when you become real developer who loves GitHub. 
Thanks for your reading.
Well, let's start discussion!

**Comment by happysmile0227** [comment]
Thanks, @PurpleMonkey729 !
I wanna know the method how to get quick draw badge.
Could you give me more detailed description?

**Reply by PurpleMonkey729** [reply]
Thanks, for your attention of this discussion.
If you need the quick draw badges, please conform the following link.
https://github.com/orgs/community/discussions/20645
Best regards!

**Comment by happysmile0227** [comment]
Your reply is very useful for me.
Thanks!
